commands/auth_utils.go

```diff
@@ -138,18 +138,9 @@ func SaveCredentialsList(credentialsList *CredentialsList) error {
 		return err
 	}
 
-	if _, err := os.Stat(configPath); err != nil {
-		if !os.IsNotExist(err) {
-			return err
-		}
-		if err := os.MkdirAll(configPath, 0700); err != nil {
-			return errors.WithMessage(err, "cannot create config directory")
-		}
-	}
-
 	marshaledCredentialsList, _ := json.MarshalIndent(credentialsList, "", "    ")
 
-	if err := ioutil.WriteFile(filepath.Join(configPath, configFileName), marshaledCredentialsList, 0600); err != nil {
+	if err := ioutil.WriteFile(configPath, marshaledCredentialsList, 0600); err != nil {
```

Comment:
Also, we can possibly extract the permission as constant, this an the 0700, to make it more readable

Reply:
I am a little bit skeptic about this. We are not using those permissions elsewhere, maybe adding a comment would be more convenient?

Reply:
I'm 0/5 in this. I like to avoid having magic words and or use this to make it more readable for another developer that wants to know which permission is 600 instead of thinking on the bytes. Up to you on this, is just a nit